Kentico/kontent-starter-corporate-next-js
# Upgrade Kontent JS SDK

### Motivation

New Kontent DAPI JS is in preview. Let's prepare this application for new version.

Primarily to avoid paring the raw data and hydrating them back to classes in the component lifecycle: https://github.com/Kentico/kontent-starter-corporate-next-js/blob/f3a194fb22b68acefc1d6a75d794208bdbd41a39/README.md#specific-route-data

### Checklist


### How to test

Verify against:
Upgrade instructions: https://github.com/Kentico/kontent-delivery-sdk-js/blob/b9d5187a6f86eb0fbff149b096127bf44cd18417/readme.md#upgrade

+ click through the preview deployment
